### PR TITLE
Add some guidance for contributors.

### PR DESCRIPTION
**Pull request by stevejb71**
I've added some extra information for contributors, to address issue #17.

Please take a look, and let me know if there is anything else that can be added, or anything that can be better worded!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,49 @@ Exercism Exercises in OCaml
...
+## Adding an Exercise
+
+The [contributing guide](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md) provides guidance on
+how to add a new exercise, or port an existing exercise from another language track.
+
+A sample exercise folder layout is below. Makefile and .merlin can be copied from any of the other exercises.
+
+```
+└── exercises
+    └── hello-world
```
Comment (dvberkel) [inline README.md]: Maybe it is better to name this `sample`, or `SAMPLE` and mention that it should be named according to the exercise slug.

Comment (stevejb71) [inline README.md]: Agreed, that would be better. I'll fix it up once I'm near my PC.

**Comment by stevejb71** [comment]
@dvberkel: I've followed your suggestion, and also added a few more updates in the second half of the document.

**Comment by dvberkel** [comment]
Awesome!

I am merging this

**Pull request merged**